### DISCUSSION TITLE
Why OboeTester can print Android system  ALOGV but my app cannot

### DISCUSSION THREAD

**Discussion by xzhan96** [google/oboe#1895] [General]
Hi @philburk 

OboeTester can print android system ALOG, such as

> 2023-08-14 15:30:59.421  1357-13911 AAudioServ...dpointMMAP audioserver                          D  createMmapBuffer() createMmapBuffer() buffer_size = 3072 fr, burst_size 48 fr, Sharable FD: Yes
> 2023-08-14 15:30:59.421  1357-13911 AAudioServ...dpointMMAP audioserver                          D  openWithFormat() got rate = 48000, channels = 2 channelMask = 0x3, deviceId = 17, capacity = 3072
> 2023-08-14 15:30:59.421  1357-13911 AAudioServ...dpointMMAP audioserver                          D  openWithFormat() got format = 0x1 = AUDIO_FORMAT_PCM_16_BIT, frame size = 4, burst size = 48

However, my app(use oboe do audio capture/playback) can't print that even in the same android device with the same oboe parameters, is there any special build settings in OboeTester? I didn't find any.

Thanks a lot for your answer!

**Comment by philburk** [comment]
Oboe defines LOG* macros in the file OboeDebug.h
https://github.com/google/oboe/blob/main/src/common/OboeDebug.h

It calls __android_log_print()
